api/v1beta1/disruption_cron_types.go

```diff
@@ -83,4 +83,14 @@ type DisruptionCronStatus struct {
 	// Time when the target resource was previously missing.
 	// +nullable
 	TargetResourcePreviouslyMissing *metav1.Time `json:"targetResourcePreviouslyMissing,omitempty"`
+
+	History []DisruptionCronTrigger `json:"history,omitempty"`
+}
+
+const MaxHistoryLen = 5
+
+type DisruptionCronTrigger struct {
+	Name      string      `json:"name,omitempty"`
+	Kind      string      `json:"kind,omitempty"`
```

Comment:
This `kind` seems to always be the kind of the crd we're reconciling, disruptioncron. Why reiterate it here? This used to store the names of the last disruption runs, now it just stores the `lastScheduledTime`s for a given cron. What's the benefit of storing name/kind here?

Reply:
https://github.com/DataDog/chaos-controller/pull/817/files#diff-67195dd0070673a7c0f061485aa4c43739add706be74ecb8fd11ff345ddaab5bR166-R168 seems different than the pr description

Reply:
what I did here is to take a snapshot of the `disruptionCron` and store in `history`. I'm unsure if `name` and `kind` of a `disruptionCron` changes are allowed in chaos-controller, but lets assume they are, then the `history` should still show the original `name, kind` at that point in time.

Reply:
Understood, :thumbsup:

Reply:
i did look it up, and can confirm that name and kind can never be mutated